bin/mainflux-install.sh

```diff
@@ -57,8 +59,15 @@ External dependencies needed for Mainflux are:
 - NATS
 - NGINX
 
-NATS have been installed.
+NATS has been installed.
 For Cassandra follow the instructions at http://cassandra.apache.org/download/
+
+After installing Cassandra you should create the two keypspaces that Mainflux uses:
+
+cqlsh> CREATE KEYSPACE IF NOT EXISTS manager WITH REPLICATION = { 'class' : 'SimpleStrategy', 'replication_factor' : 1 };
```

Comment:
Please note that this replication factor is acceptable either for "quick-and-dirty" demo deployments, or for constrained environments. This should be noted in the docs as well.